client/src/components/ModalAddFlyingSite.vue

```diff
@@ -154,6 +155,27 @@ const saveButtonIsEnabled = computed(() => {
   );
 });
 
+const Toast = Swal.mixin({
+  toast: true,
+  position: "top-end",
+  showConfirmButton: false,
+  timer: 3000,
+  timerProgressBar: true,
+});
+
+const addSiteSuccess = () => {
+  Toast.fire({
+    icon: "success",
+    title: "Dein Vorschlag wurde übermittelt",
+  });
+};
+const addSiteFailed = () => {
```

Comment:
Ich hab eigentlich immer vermieden das in einem Modal zu verwenden. Den Success indicator vielleicht noch ja, denn das Modal verschwindet ja. Aber den Error würde ich wie an allen anderen Stellen mit GenericError direkt im Modal anzeigen.

Reply:
Den für failed habe ich entfernt und einen error text ins modal gepackt.

Wollte gerne noch einen weiteren Indicator, da das Modal halt schließt und man sonst kein "richtiges" Feedback über die Aktion bekommt.

Reply:
Ja der success geht ja auch voll klar.